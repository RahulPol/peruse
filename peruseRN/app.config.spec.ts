import config from './app.config';

describe('app.config', () => {
  it('should match config snapshot', () => {
    expect(config).toMatchSnapshot();
  });
});
