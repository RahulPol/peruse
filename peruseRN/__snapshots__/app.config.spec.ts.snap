// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app.config should match config snapshot 1`] = `
Object {
  "android": Object {
    "adaptiveIcon": Object {
      "backgroundColor": "#FFFFFF",
      "foregroundImage": "./assets/icon_2.png",
    },
    "package": "com.peruse.peruseRN",
    "softwareKeyboardLayoutMode": "pan",
  },
  "icon": "./assets/logo_2.jpg",
  "name": "PeruseRN",
  "orientation": "portrait",
  "slug": "PeruseRN",
  "splash": Object {
    "backgroundColor": "#ffffff",
    "image": "./assets/logo_2.jpg",
    "resizeMode": "contain",
  },
  "version": "0.0.1",
}
`;
